pennylane/__init__.py

```diff
@@ -45,7 +45,7 @@
 from pennylane.optimize import *
 from pennylane import qaoa
 from pennylane.qnode import QNode, qnode
-from pennylane.templates import broadcast, layer, template
+from pennylane.templates import broadcast, layer, template, QFT
```

Comment:
I don't think you should import QFT here as it should be included in template?

Reply:
I thought so too, but many places in the code QFT is being used as pennylane.QFT directly...and thus those places the code fails, I thought instead of adding pennylane.templates.QFT everywhere just import it here , I also tried 'from templates import *' in this init file similar to how ops is imported...but I got many more other errors..

Reply:
You have a good point, I'll come back to you 👍

Reply:
I spoke with my colleague and we prefer the solution with `qml.templates.QFT`otherwise QFT would be the only template working differently. That only means to replace `qml.QFT` by `qml.templates.QFT`. Thank you 👍

Reply:
cool I'll change it then